app/assets/javascripts/components/activity/recent_activity_handler.cjsx

```diff
@@ -10,16 +10,21 @@ RecentEditsHandler = require './recent_edits_handler'
 RecentActivityHandler = React.createClass(
   displayName: 'RecentActivityHandler'
   render: ->
+
+    dIYEligible = I18n.t('recent_activity.did_you_know_eligible')
```

Comment:
works for me! I didn't realize I could render the i18n calls by just sticking in a handlebar, but of course that makes sense if the handlebars can render a variable assigned to them.